site/docs/spark-procedures.md

```diff
@@ -221,12 +222,13 @@ Used to remove files which are not referenced in any metadata files of an Iceber
 
 #### Usage
 
-| Argument Name | Required? | Type | Description |
-|---------------|-----------|------|-------------|
-| `table`       | ✔️  | string | Name of the table to clean |
-| `older_than`  | ️   | timestamp | Remove orphan files created before this timestamp (Defaults to 3 days ago) |
-| `location`    |    | string    | Directory to look for files in (defaults to the table's location) |
-| `dry_run`     |    | boolean   | When true, don't actually remove files (defaults to false) |
+| Argument Name | Required? | Type      | Description                                                                                                                                                |
+|--------|-----------|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Seems some unnecessary ```---``` got removed, though I guess as it's MD table it's not a big deal

Reply:
Oh I can add it back in though. My bad!

Reply:
Hmmm..... looking at it closer though, it does kind of appear like maybe it was trying to be lined up at one point? I don't know what the standards are here, but probably it was just another IntelliJ user like me that had it formatted the way it was.